### PR TITLE
onboard toolchain-operator to use openshift ci

### PR DESCRIPTION
**Pull request by dipak-pawar**
To enable CodeReady Toolchain on any Openshift cluster, we need one time configuration setup which is done by toolchain-operator.

This PR onboards toolchain-operator to use openshift CI

### CODE REVIEW

File: ci-operator/config/fabric8-services/toolchain-operator/fabric8-services-toolchain-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+
+binary_build_commands: make build
+
+tag_specification:
+  name: "4.0"
```
Comment (stevekuznetsov) [inline ci-operator/config/fabric8-services/toolchain-operator/fabric8-services-toolchain-operator-master.yaml]: 4.1?

Comment (dipak-pawar) [reply ci-operator/config/fabric8-services/toolchain-operator/fabric8-services-toolchain-operator-master.yaml]: Good point @stevekuznetsov. Update it

**Comment by dipak-pawar** [comment]
@stevekuznetsov, @smarterclayton This PR is ready for review. Can you please look into it when you have time?

**Comment by droslean** [comment]
/hold cancel

If you will need to `promote` your images, it can be done in a follow-up PR.

**Pull request merged**